src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -486,6 +482,11 @@ private void processPackage(DetailAST ast, DetailAST nextToken) {
         if (!hasEmptyLineAfter(ast)) {
             log(nextToken.getLineNo(), MSG_SHOULD_BE_SEPARATED, nextToken.getText());
         }
+        // workaround of antlr bug, it puts comments into type definition modifiers
```

Comment:
it is not a bug, it is by design.
please change comment or remove it.

Reply:
done